kotlin/display-device-location/src/main/java/com/esri/arcgisruntime/sample/displaydevicelocation/SpinnerAdapter.kt

```diff
@@ -0,0 +1,63 @@
+package com.esri.arcgisruntime.sample.displaydevicelocation
+
+/* Copyright 2020 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import android.app.Activity
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ArrayAdapter
+import android.widget.ImageView
+import android.widget.TextView
+import kotlinx.android.synthetic.main.spinner_layout.view.*
+import java.util.ArrayList
+
+
```

Comment:
to comply with KDoc, we should include a brief (one - two line) description of what this class is doing

Reply:
I'll keep that in mind for the future as well!